etc/default/grub.d/45_debug-misc.cfg

```
@@ -10,13 +10,15 @@
 
 GRUB_CMDLINE_LINUX_DEFAULT="$(echo "$GRUB_CMDLINE_LINUX_DEFAULT" | sed 's/quiet//g')" || echo "$0: Removing 'quiet' from GRUB_CMDLINE_LINUX_DEFAULT failed! Please report this bug!" >&2
 
+GRUB_CMDLINE_LINUX_DEFAULT="$(echo "$GRUB_CMDLINE_LINUX_DEFAULT" | sed 's/loglevel=0//g')" || echo "$0: Removing 'loglevel=0' from GRUB_CMDLINE_LINUX_DEFAULT failed! Please report this bug!" >&2
+
 GRUB_CMDLINE_LINUX_DEFAULT="$(echo "$GRUB_CMDLINE_LINUX_DEFAULT" | sed 's/rhgb//g')" || echo "$0: Removing 'rhgb' from GRUB_CMDLINE_LINUX_DEFAULT failed! Please report this bug!" >&2
 
-GRUB_CMDLINE_LINUX_DEFAULT="$(echo "$GRUB_CMDLINE_LINUX_DEFAULT" | sed 's/loglevel=0//g')" || echo "$0: Removing 'loglevel=0' from GRUB_CMDLINE_LINUX_DEFAULT failed! Please report this bug!" >&2
+GRUB_CMDLINE_LINUX_DEFAULT="$(echo "$GRUB_CMDLINE_LINUX" | sed 's/debugfs=off//g')" || echo "$0: Removing 'debugfs=off' from GRUB_CMDLINE_LINUX failed! Please report this bug!" >&2
```

There are a few places where the wrong variable is being overwritten.

```suggestion
GRUB_CMDLINE_LINUX="$(echo "$GRUB_CMDLINE_LINUX" | sed 's/debugfs=off//g')" || echo "$0: Removing 'debugfs=off' from GRUB_CMDLINE_LINUX failed! Please report this bug!" >&2
```